### DISCUSSION TITLE
64bit wget

### DISCUSSION THREAD

**Discussion by winterz** [chocolatey-community/chocolatey-packages#1822] [Ideas]
The wget package provides a 32bit version of the executable thereby limiting the maximum download size to 2GB.
There is a 64-bit version at https://eternallybored.org/misc/wget/
can we use that instead?

**Comment by TheCakeIsNaOH** [comment]
What version of the wget package are you running?

Because the latest version of the package (`1.21.3`) does include both the 32 and 64 bit archives from the eternallybored site:
https://community.chocolatey.org/packages/wget/1.21.3#files

Show the `VERIFICATION.txt` for details for the original download links for the included archives.

And I just reinstalled the package, and can confirm it does extract the 64 bit archive for me: 
```
Wget v1.21.3 (forced)
wget package files install completed. Performing other installation steps.
Getting x64 bit zip
Extracting wget-1.21.3-win64_x64.zip to C:\ProgramData\chocolatey\lib\Wget\tools...
C:\ProgramData\chocolatey\lib\Wget\tools
````

**Comment by winterz** [comment]
indeed. seems good.
I just successfully downloaded a 2.64Gb file
